tools/mgko/tools.go

```diff
@@ -28,76 +25,19 @@ func Command(ctx context.Context, args ...string) *exec.Cmd {
 	return mgtool.Command(commandPath, args...)
 }
 
-func PublishLocalCommand(ctx context.Context) *exec.Cmd {
```

Comment:
Doesn't it make sense to have these convenience wrappers here? This is a lot of stuff to remember how to write :sweat_smile:

Reply:
They are too specific and won't generalize well to all our repos - better to have them inlined IMO

Reply:
Fair enough!